Gemfile

```diff
@@ -105,4 +105,5 @@ group :test do
   gem 'mutant-rspec',  github: 'mbj/mutant', ref: '90d103dc323eded68a7e80439def069f18b5e990'
   gem 'selenium-webdriver'
   gem 'simplecov', require: false
+  gem 'timecop'
```

Comment:
Can we use `ActiveSupport::Testing::TimeHelpers`?

https://api.rubyonrails.org/v6.0/classes/ActiveSupport/Testing/TimeHelpers.html

We used to use timecop, but have found it isn't typically needed anymore.

Reply:
Cool, I didn't know about this! Updated